### PR TITLE
Skip husky install in CI environments

### PR DESCRIPTION
**Pull request by eddieferrer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
 		"storybook": "start-storybook -p 6006",
 		"build-storybook": "build-storybook",
 		"deploy-storybook": "storybook-to-ghpages",
-		"prepare": "husky install"
+		"prepare": "if [[ !${CI} != *true ]]; then husky install; fi"
```
Comment (mcstover) [inline package.json:37]: Do we need to set this somewhere?

File: package.json [hunk 2]
```diff
@@ -167,6 +166,7 @@
 		"git-revision-webpack-plugin": "^3.0.6",
 		"hard-source-webpack-plugin-fixed-hashbug": "^0.13.2",
 		"html-minifier": "^4.0.0",
+		"husky": "^5.2.0",
```
Comment (mcstover) [inline package.json:169]: did you run npm i again after moving this to update package-lock

Comment (eddieferrer) [reply package.json:169]: Whoops! Will do

File: package.json (hunk 1 above)
Comment (eddieferrer) [reply package.json:37]: It's set in `Jenkinsfile`

Comment (mcstover) [reply package.json:37]: oh indeed!

**Review by mcstover — APPROVED** [review]

**Pull request merged**